### DISCUSSION TITLE
[A-L] (2023/24) Foglio 5 - Esercizio 8

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#311] [Esercizi - Viaggi/Piazza]
![Screenshot 2023-11-04 181422](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/94874fab-74e1-4c89-8ef6-d9ab1483255f)

**Comment by IonutCicio** [comment]
Sappiamo che $\mathcal{U}(\mathbb{Z_8}) = \\{ 1, 3, 5, 7 \\}$ _(tutti gli elementi coprimi con 8)_, per cui la tabella moltiplicativa è

| $\cdot$ | 1 | 3 | 5 | 7 |
|:--:|:--:|:--:|:--:|:--:|
| **1** | 1 | 3 | 5 | 7 |
| **3** | 3 | 1 | 7 | 5 |
| **5** | 5 | 7 | 1 | 3 |
| **7** | 7 | 5 | 3 | 1 |

Ora, grazie all'esercizio 3, si ha che $\phi$ isomorfo $\implies o(g) = o(\phi(g)), \forall g \in G$. Solo per essere più pedanti:

$$
\phi : (\mathcal{U}(\mathbb{Z}_8), \cdot) \to (\mathbb{Z}_4, +)
$$

In $\mathcal{U}(\mathbb{Z}_8)$ si ha $o(1) = 1, o(3) = 2, o(5) = 2, o(7) = 2$
In $\mathbb{Z}_4$ si ha $o(0) = 1, o(1) = 4, o(2) = 2, o(3) = 4$

Visto che stiamo considerando il gruppo $(\mathbb{Z}_4, +)$, si ha per definizione che $z \in \mathbb{Z}_4, z^t = (z + z + ... + z) \\ t \text{ volte} = z \cdot t$

Quindi si ha che
- $0 \cdot 1 \equiv_4 0 \implies o(0) = 1$ ($0$ è l'elemento neutro in $\mathbb{Z}_4$)
- $1 \cdot 4 \equiv_4 0 \implies o(1) = 4$
- $2 \cdot 2 \equiv_4 0 \implies o(2) = 2$
- $3 \cdot 4 \equiv_4 0 \implies o(3) = 4$

I due gruppi non sono isomorfi.

**Reply by Anastasiyan13** [reply]
![IMG_0151](https://github.com/sapienzastudentsnetwork/algebra2324/assets/126688724/ffbdebc0-3b1f-4d20-9482-6c51354b25a2)

**Reply by IonutCicio** [reply]
Hai ragione, mia svista, non avevo compreso bene l'esercizio quando lo feci al tempo. Provvedo a corregere, grazie per avermelo fatto notare 🫡

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/e4afef8d-b7ef-4ca4-83fc-b967d3bfea10)

**Reply by dahKidou** [reply]
Scusami, non mi è chiara una cosa, sicuramente per mie carenze teoriche. Perchè nella prima soluzione sopra o(g) = 1 per ogni g appartenente agli invertibili di Z8, ma nella sol. alternativa o(3) = o(5) = o(7) = 2 (dato che 3, 5 e 7 sono elementi di Z8) ?

**Reply by CiottoloMaggico** [reply]
Era un errore della prima soluzione, credo di aver fatto un po' di confusione con gli ordini. Fai riferimento a quella che c'è ora
